src/pilot/dell_nfv.py

```diff
@@ -211,62 +173,66 @@ def edit_environment_files(
-                '|" ' +
-                file_path)
-
+            if not ovs_dpdk:
+                cmds.append('sed -i "s|  # NovaSchedulerDefaultFilters|  ' +
+                            'NovaSchedulerDefaultFilters|" ' + file_path)
             cmds.append(
                 'sed -i "s|DellComputeCount:.*|DellComputeCount: ' +
                 str(dell_compute_count) +
                 '|" ' +
                 file_path)
 
             if enable_hugepage:
-                hugepage_number = ConfigOvercloud.calculate_hugepage_count(
+                hpg_num = ConfigOvercloud.calculate_hugepage_count(
                     hugepage_size)
-                cmds.append('sed -i "s|dellnfv::hugepages::hugepagesize:.*'
-                            '|dellnfv::hugepages::hugepagesize: ' +
-                            hugepage_size[0:-1] + '|" ' + file_path)
-                cmds.append(
-                    'sed -i "s|dellnfv::hugepages::hugepagecount:.*'
-                    '|dellnfv::hugepages::hugepagecount: ' +
-                    str(hugepage_number) +
-                    '|" ' +
-                    file_path)
+                hugecmd = 'default_hugepagesz=' + \
+                    hugepage_size + ' hugepagesz=' + \
+                    hugepage_size[0:-1]+' hugepages=' + \
+                    str(hpg_num)+' iommu=pt intel_iommu=on'
+                if not ovs_dpdk:
+                    cmds.append('sed -i "s|HugepagesEnable.*|' +
+                                'HugepagesEnable: true|" ' +
+                                file_path)
+                    cmds.append("sed -i 's|HugePages:.*|HugePages: \"" +
+                                hugecmd + "\"|' " + file_path)
+
             if enable_numa:
-                vcpu_pin_set = ConfigOvercloud.calculate_hostos_cpus(
-                    hostos_cpu_count)
-                cmds.append(
-                    "sed -i 's|dellnfv::numa::vcpu_pin_set:.*"
-                    "|dellnfv::numa::vcpu_pin_set: \"" +
-                    vcpu_pin_set +
-                    "\"|' " +
-                    file_path)
-                cmds.append(
-                    'sed -i "s|  # NovaVcpuPinSet|  ' +
-                    'NovaVcpuPinSet|" ' +
-                    file_path)
+                ConfigOvercloud.calculate_hostos_cpus(hostos_cpu_count)
+                if not ovs_dpdk:
+                    cmds.append('sed -i "s|NumaEnable:.*|NumaEnable: true|" ' +
+                                file_path)
+                    cmds.append(
+                        "sed -i 's|NumaCpus:.*|NumaCpus: " +
+                        VCPUS +
+                        "|' " +
+                        file_path)
+                    cmds.append(
+                        'sed -i "s|  # NovaVcpuPinSet|  ' +
+                        'NovaVcpuPinSet|" ' +
+                        file_path)
+                    cmds.append(
+                        "sed -i 's|NovaVcpuPinSet:.*|NovaVcpuPinSet: \"" +
```

Comment:
I don't see this option within the dell-environment.yaml file. This should present and commented as default. If numa is enabled, then this command will succeed.

Reply:
It is there 
https://github.com/uday344/JetPack/blob/cdf64a095075b864073bf5a686b78fc01bbd81b5/src/pilot/templates/dell-environment.yaml#L30

Reply:
gotcha.